notebooks/GWAS_1000_genomes_p1/main.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Notebook author: Beth Sheets
+# Herzog version: Ash O'Farrell
+
+# The notebook didn't have a markdown title at the time but I've dubbed it Phenotype Consolidator.
+# The name of the actual file is 1-Prepare-Gen3-Data-For-Exploration
+
+import os
+import herzog
+
+# Mock the environment
+os.environ['WORKSPACE_NAME'] = "CICD TESTER - PUBLIC DATA ONLY - BDC GWAS 1000G Tutorial"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-7d1224a1-1770-4db9-9904-137efcd412c5"
+os.environ['GOOGLE_PROJECT'] = "anvil-stage-demo"
+
+with herzog.Cell("markdown"):
+    """
+    # Phenotype Consolidater
+    *version: 2.0*
+
+    ## Notebook Overview
+
+    This notebook consolidates the phentotypic data in tables imported to Terra from Gen3 into one consolidated table of metadata with familiar subject IDs for downstream analysis. It uses functions defined in the terra-util notebook, and can be adapted to consolidate different metadata.
+
```

Comment:
One big piece of feedback we could consider from fellows is to break this notebook up into two pieces: one for phenotype analysis using a standard VM, and one for genotype analysis using the hail container and a spark cluster. This may reduce cost for people that are working through it slowly... we could delay this to future work if that's a big request.

Reply:
My understanding is that this is already the case. There's two notebooks in this repo, one that consolidates the phenotypic tables on a standard VM, and another that does genotypic analysis with Hail + Spark. The second notebook is quite long, though, is that the one that you want to split?

Reply:
* two notebooks in this pull request

Reply:
My feedback was to split "2-GWAS-initial-analysis" into two parts, what could be: "2-phenotypic-analysis", "3-genetic structure".

"2-phenotypic-analysis" would import the consolidated phenotypic data table in, subset, and explore phenotypes to select and the output would be "my_phenotypes.csv". 

"3-genetic-structure" would import my_phenotypes.csv back in and start a hail session to do all of the pop gen analysis. 

We could work on this together if you want.

Reply:
I made a Jira ticket to do this in a future sprint. For now, if Lon and Brian are okay with it, I'm ready to PR the two notebooks as-is then do the split.